modules/opencensus/src/test/java/org/apache/ignite/internal/processors/monitoring/opencensus/OpenCensusDiscoveryTracingTest.java

```diff
@@ -116,7 +116,8 @@ public void testClientNodeCacheCreateTracing() throws Exception {
                 catch (Exception e) {
                     fail("Unable to create cache.");
                 }
-            }
+            },
+            GRID_CNT
```

Comment:
As far as I understand it was made in order to overcome the problem of configuration unavailability on client node, cause of "async" nature of metastore data propagation to the client nodes.
Despite the fact that it's valid to use same node for both setting configuration and trigger operation to be traced, I'd rather use wait for condition approach within some utility method. What's more important we should ensure that any of given approaches is used everywhere but not only within given test. For example OpenCensusTxTracingTest also should be modified.
In other words I suggest to add some sort of `ensureTracingConfigurationWasSet(expectedConfig, cluster) ` that will be triggered after setting configuration and starting client nodes.

Reply:
Ok, it makes sense to me. I will try to modify tests in that way.